ansible/roles/ocp-workload-vertx-reactica/defaults/main.yml

```diff
@@ -0,0 +1,23 @@
+---
+become_override: false
```

Comment:
can you also add a default for `namespace`, for example

```yaml
namespace: "reactica-{{guid}}"
```
?
leaving `ocp-username` and `guid` the only variables to pass